core/views.py

```diff
@@ -6,15 +6,23 @@
 from django.contrib.auth.mixins import LoginRequiredMixin, PermissionRequiredMixin
 from django.contrib.contenttypes.models import ContentType
 from django.core.exceptions import ObjectDoesNotExist
+<<<<<<< HEAD
```

Comment:
To resolve these import conflicts, I'd suggest to keep the imports from both your branch and those from master.